server/test/test.js

```diff
@@ -0,0 +1,13 @@
+const assert = require('assert');
+const firebase = require('@firebase/testing'); 
+
+const PROJECT_ID = "step9-2020-capstone"; 
+
+describe("Our political ads app", () => {
+  it("Can read items in read-only collection", async() => {
```

Comment:
Other things to test: an ad uploaded from your java is read into firebase correctly